raiden/tests/unit/test_matrix_presence.py

```diff
@@ -11,23 +11,9 @@
 from raiden.utils import Address
 
 
-class DummyUser:
```

Comment:
This change is explained in the commit. tl;dr The change to get_user forced the use of the `User` class which made the mapping lookup unstable, because `User` doesn't have `__hash__`, the fix was to use the `user_id` instead of `DummyUser` as a key for the test's presence mapping.